### PR TITLE
Dots that represent items inside closed sections; Fix pages scroll

### LINKED ISSUE #1262: Show dots that represent items inside closed sections in style panel
Split from #1100

Screenshot 2023-03-17 at 15 30 06

@taylornowotny @kof some questions:

 - Do we want the dots only for sections like "Backgrounds" where you add/remove items, or for all sections?
 - Do we want to show `preset` as a dot? Currently in Figma color for the `preset` dot is not defined.

### LINKED ISSUE #1308: There's no scroll in pages list
https://discord.com/channels/955905230107738152/1090138811897499698

### PR DESCRIPTION
**Pull request by rpominov**
## Description

1. Fixes #1308
2. Closes #1262

## Steps for reproduction

1. Modify some styles
3. Close section, notice the dots

## Code Review

- [ ] hi @kof, I need you to do
  - conceptual review (architecture, feature-correctness)
  - test it on preview
- [ ] hi @istarkov, I need you to do
  - have a look at whether backgrounds work as expected

## Before requesting a review

- [ ] made a self-review
- [ ] added inline comments where things may be not obvious (the "why", not "what")

## Before merging

- [ ] tested locally and on preview environment (preview dev login: 5de6)
- [ ] updated [test cases](https://github.com/webstudio-is/webstudio-builder/blob/main/apps/builder/docs/test-cases.md) document
- [ ] added tests
- [ ] if any new env variables are added, added them to `.env.example` and the `builder/env-check.js` if mandatory

### CODE REVIEW

File: apps/builder/app/builder/features/style-panel/shared/style-collapsible-section.tsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import { SectionTitle, SectionTitleLabel } from "@webstudio-is/design-system";
+import type { ReactNode } from "react";
+import {
+  CollapsibleSectionBase,
+  useOpenState,
+} from "~/builder/shared/collapsible-section";
+import type { StyleSource } from "./style-info";
+
+export const StyleCollapsibleSection = (props: {
```
Comment (kof) [inline apps/builder/app/builder/features/style-panel/shared/style-collapsible-section.tsx:9]: Style as in StylePanelCollapsibleSection? I think we could just call it CollapsibleSection and rely on the import

**Comment by kof** [comment]
I am not sure if related but it seems that remote property label is not displayed any more https://share.descript.com/view/NQDNV8b5Fxu

**Review by kof — APPROVED** [review]

**Comment by rpominov** [comment]
> I am not sure if related but it seems that remote property label is not displayed any more https://share.descript.com/view/NQDNV8b5Fxu

Haven't touched this

**Review by istarkov — APPROVED** [review]

File: apps/builder/app/builder/features/style-panel/style-settings.tsx [hunk 2]
```diff
@@ -148,8 +145,9 @@ export const StyleSettings = ({
         category,
       };
 
-      // @todo remove isRendered once space section is converted to a section
-      if (isInCategory && isApplicable && isRendered === false) {
+      if (isInCategory && isApplicable) {
```
[the remark this answers is not in the document]
Comment (istarkov) [reply apps/builder/app/builder/features/style-panel/style-settings.tsx:148]: I would calc "sources" per section individually, and somehow avoid that config categoryProperties in the future or at least make section expose it

[the remark this answers is not in the document]
Comment (rpominov) [reply apps/builder/app/builder/features/style-panel/style-settings.tsx:148]: Will make sure they're in sync at the moment. 

And to make it easier to keep them in sync in the future, I would co-locate list of properties with implementations of sections.

@kof What do you think?

**Review by kof — APPROVED** [review]

**Pull request merged**